src/main/kotlin/platform/architectury/version/ArchitecturyVersion.kt

```diff
@@ -0,0 +1,116 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2021 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.platform.architectury.version
+
+import com.demonwav.mcdev.util.MinecraftVersions
+import com.demonwav.mcdev.util.SemanticVersion
+import java.io.IOException
+import java.net.URL
+import javax.xml.stream.XMLInputFactory
+import javax.xml.stream.events.XMLEvent
+
+class ArchitecturyVersion private constructor(val versions: List<String>) {
+
+    fun getArchitecturyVersions(mcVersion: SemanticVersion): List<SemanticVersion> {
+        return versions.asSequence()
+            .filter {
+                it.startsWith(
+                    when {
+                        mcVersion < MinecraftVersions.MC1_17 -> "1"
+                        mcVersion >= MinecraftVersions.MC1_17 && mcVersion < MinecraftVersions.MC1_18 -> "2"
+                        mcVersion >= MinecraftVersions.MC1_18 -> "3"
+                        else -> "0"
```

Comment:
If there is an API for this, it would be better to use that so we don't have to maintain this list

Reply:
Short of making a nodejs rest api for parsing the mc version off of the forge jar file, I see no good way of doing this. I don’t want to have to download the jar file and load it into the classpath at runtime and use reflection…

Reply:
Although that rest api could be useful for other people too so I might do it :)

Reply:
It might be best to ask architectury to provide such an API themselves

Reply:
I’ve decided. I’m making that rest api because it’s cool and useful. And it would clean this but up here real nice.

Reply:
It really would be better to have an official API, for a plugin that so many people use it would be more reliable. I've asked about it in their discord.

Reply:
So have I

Reply:
For future reference and so I don’t forget,
Here’s the link to the json:
https://gist.githubusercontent.com/shedaniel/4a37f350a6e49545347cb798dbfa72b3/raw/architectury.json